components/match2/commons/match_group_util.lua

```diff
@@ -409,6 +409,32 @@ function MatchGroupUtil.bracketDataFromRecord(data)
 	end
 end
 
+function MatchGroupUtil.bracketDataToRecord(bracketData)
+	local coordinates = bracketData.coordinates
+	return {
+		bracketreset = bracketData.bracketResetMatchId,
+		coordinates = coordinates and MatchGroupUtil.indexTableToRecord(coordinates),
+		header = bracketData.header,
+		loweredges = bracketData.lowerEdges and Array.map(bracketData.lowerEdges, MatchGroupUtil.indexTableToRecord),
+		lowerMatchIds = bracketData.lowerMatchIds,
+		qualwinLiteral = bracketData.qualwinLiteral,
+		quallose = bracketData.qualLose and 'true' or nil,
+		qualloseLiteral = bracketData.qualLoseLiteral,
+		qualskip = bracketData.qualSkip ~= 0 and bracketData.qualSkip or nil,
+		qualwin = bracketData.qualWin and 'true' or nil,
+		skipround = bracketData.skipRound ~= 0 and bracketData.skipRound or nil,
+		thirdplace = bracketData.thirdPlaceMatchId,
+		type = bracketData.type,
+		upperMatchId = bracketData.upperMatchId,
+
+		-- Deprecated
+		bracketsection = coordinates
```

Comment:
Since when? This is in active use.

Reply:
It's not used in the lua code, so I assume it's used by some other project? My intention was to subsume it under `match2bracketdata.coordinates.sectionIndex`. It's not important though. I'm fine with keeping it part of the API not deprecated if you like.

Reply:
I'm open to changing it, but yes it's in heavy use internally (which goes for almost all bracketdata properties FWIW). So let's leave that out of this PR. If you still want to change it, sure (`sectionIndex` definitely sounds better) but will need breaking change notifications etc